packages/pigment-css-react/src/processors/css.ts

```diff
@@ -9,7 +9,7 @@ import type {
 } from '@wyw-in-js/processor-utils';
 import type { Replacements, Rules } from '@wyw-in-js/shared';
 import { ValueType } from '@wyw-in-js/shared';
-import type { CSSInterpolation } from '@emotion/css';
+import type { CSSInterpolation } from '@emotion/serialize';
```

Comment:
How did this breaking change land in an minor 😕

Reply:
Yeah. I was surprised as well.